packages/core/src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -84,6 +93,82 @@ export class AbstractQueryInterfaceTypeScript {
     return schemaNames.flatMap((value: any) => (value.schema_name ? value.schema_name : value));
   }
 
+  /**
+   * Describe a table structure
+   *
+   * This method returns an array of hashes containing information about all attributes in the table.
+   *
+   * ```js
+   * {
+   *    name: {
+   *      type:         'VARCHAR(255)', // this will be 'CHARACTER VARYING' for pg!
+   *      allowNull:    true,
+   *      defaultValue: null
+   *    },
+   *    isBetaMember: {
+   *      type:         'TINYINT(1)', // this will be 'BOOLEAN' for pg!
+   *      allowNull:    false,
+   *      defaultValue: false
+   *    }
+   * }
+   * ```
+   *
+   * @param tableName
+   * @param options Query options
+   *
+   */
+  async describeTable(tableName: TableNameOrModel, options?: DescribeTableOptions) {
```

Comment:
this is missing a return type

Reply:
I haven't specified a return type as setting the options for the `queryRaw` to a value with type the below currently returns a type as per:


https://github.com/sequelize/sequelize/blob/0a572b901b6aaef438f127a99f7754ca3955473c/packages/core/src/sequelize.d.ts#L956-L956

```typescript
const queryOptions: QueryRawOptionsWithType<QueryTypes.DESCRIBE> = { ...options, type: QueryTypes.DESCRIBE };
```

If you want I can specify it to make sure its returned.

Reply:
I'd prefer to set them explicitly in places like this one :)

Reply:
Good point. I've added this back in now.